js/signin.js

```diff
@@ -1,118 +1,72 @@
-const querySelector = (selector) => document.querySelector(selector);
+import { emailInput, passwordInput, emailErrorMessage, passwordErrorMessage, loginForm, eyeOffButton, eyeOnButton } from "./elements.js";
+import { isValidEmail, showErrorMessage, hideErrorMessage, togglePasswordVisibility } from "./sign.js";
 
-const emailInput = querySelector("#email-input");
-const passwordInput = querySelector("#password-input");
-const emailErrorMessage = querySelector("#email-error-message");
-const passwordErrorMessage = querySelector("#password-error-message");
-const loginForm = querySelector("#login-form");
-
-const eyeOffButton = querySelector("#eye-off");
-const eyeOnButton = querySelector("#eye-on");
-
-const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
+document.addEventListener("DOMContentLoaded", () => {
+  const accessToken = localStorage.getItem("accessToken");
+  if (accessToken) {
+    window.location.href = "/folder.html";
+  }
+});
 
 // 이메일 입력
-emailInput.addEventListener("focusout", validateEmailInput);
-
-function validateEmailInput() {
+emailInput.addEventListener("focusout", () => {
   const email = emailInput.value.trim();
-
   if (email === "") {
-    showEmailErrorMessage("이메일을 입력해 주세요.");
+    showErrorMessage(emailInput, emailErrorMessage, "이메일을 입력해 주세요.");
     return;
   }
-
-  if (!isValidateEmail(email)) {
-    showEmailErrorMessage("올바른 이메일 주소가 아닙니다.");
+  if (!isValidEmail(email)) {
+    showErrorMessage(emailInput, emailErrorMessage, "올바른 이메일 주소가 아닙니다.");
     return;
   }
-
-  hideEmailErrorMessage();
-}
-
-function isValidateEmail(email) {
-  return EMAIL_REGEX.test(email);
-}
-
-function showEmailErrorMessage(message) {
-  emailErrorMessage.textContent = message;
-  emailErrorMessage.classList.add("show");
-  emailInput.classList.add("sign-input-error");
-  emailInput.classList.add("error-focus");
-}
-
-function hideEmailErrorMessage() {
-  emailErrorMessage.textContent = "";
-  emailErrorMessage.classList.remove("show");
-  emailInput.classList.remove("sign-input-error");
-  emailInput.classList.remove("error-focus");
-}
+  hideErrorMessage(emailInput, emailErrorMessage);
+});
 
 // 비밀번호 입력
-passwordInput.addEventListener("focusout", validatePasswordInput);
-
-function validatePasswordInput() {
+passwordInput.addEventListener("focusout", () => {
   const password = passwordInput.value.trim();
-
   if (password === "") {
-    showPasswordErrorMessage("비밀번호를 입력해 주세요.");
+    showErrorMessage(passwordInput, passwordErrorMessage, "비밀번호를 입력해 주세요.");
     return;
   }
+  hideErrorMessage(passwordInput, passwordErrorMessage);
```

Comment:
요기도 사소하지만 개행이 들어가면 좋을 것 같아요!
```suggestion
  }
  
  hideErrorMessage(passwordInput, passwordErrorMessage);
```